Library/Formula/libgeotiff.rb

```diff
@@ -4,6 +4,7 @@ class Libgeotiff < Formula
   homepage 'http://geotiff.osgeo.org/'
   url 'http://download.osgeo.org/geotiff/libgeotiff/libgeotiff-1.4.1.tar.gz'
   sha1 'bc9e2bb43f3877b795b4b191e7aec6267f4a1c7e'
+  revision 1
```

Comment:
`libgeotiff` I believe, maybe everything with a new `revision` (if you check the build output)